src/main/java/fi/fmi/avi/util/BulletinHeadingDecoder.java

```diff
@@ -64,6 +68,7 @@ public static BulletinHeading decode(final String input, final ConversionHints h
                 .setBulletinNumber(Integer.parseInt(m.group("ii")))//
                 .setType(type)//
                 .setBulletinAugmentationNumber(Optional.ofNullable(bulletinAugmentationNumber))//
+                .setBulletinAugmentationIndicator(bbb)//
```

Comment:
I think that the original bulletin augmentation indicator should be set here instead of the one possibly corrected by the `BulletinHeadingIndicatorInterpreter`. Because the corrected indicator can always be reconstructed from `type`and `bulletinAugmentationNumber`. (Which actually could be a helper method in `BulletinHeading`, as this reconstruction is needed sometimes anyway. The reconstruction method might actually clarify the distinction between the reconstructed and original augmentation indicator in the API?)